contracts/KyberNetwork.sol

```diff
@@ -23,7 +23,7 @@ contract KyberNetwork is Withdrawable, Utils {
     FeeBurnerInterface    public feeBurnerContract;
     uint                  public maxGasPrice = 50 * 1000 * 1000 * 1000; // 50 gwei
     bool                  public enabled = false; // network is enabled
-    uint                  public networkState; // this is only a field for UI.
+    mapping(bytes32=>uint) public info; // this is only a UI field for external app.
```

Comment:
don't understand why bytes32? 
what key type will be used?
how will we know per query the key?

Reply:
Network state will be listed as `info(sha3("network state")), or any other way. It is in application level, not contract level.